src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Accordion as ChakraAccordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from "@chakra-ui/react"
+
+type AccordionProps = {
+  label: string
+  children: React.ReactNode
+}
+
+const Accordion = ({ label, children }: AccordionProps) => (
```

Comment:
@nloureiro do we have a defined minimum width for the accordion? Or should we define a minimum gap between the button text and the arrow icon?

Reply:
it should be the minimum gap, in this case 0.5rem

something like this?
![Screen Shot 2024-03-27 12 33 25 AM](https://github.com/ethereum/ethereum-org-website/assets/1120748/6df683ec-74d0-4f4f-a40b-5135bfb9946a)

Reply:
@nloureiro ok! Or `2` as the Chakra token value here.

I did not see this defined gap in figma; it was only being marked as "auto"

Reply:
true.... figma limitation here. It assumes a "flex box" set to auto OR a gap fixed spacing. (without making it too complex on Figma. I could have made another layer on top with the fixed gap built-in)
 on code, we can have both.
I should have done a better spec. Sorry about this...

Reply:
Oh interesting! Good to know, no problem 👍